lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -497,25 +537,43 @@ defmodule ExUnit.DocTest do
 
   defp adjust_indent(:code, [line | rest], line_no, adjusted_lines, indent, module) do
     stripped_line = strip_indent(line, indent)
+
     cond do
       stripped_line == "" ->
-        adjust_indent(:text, rest, line_no + 1, [{stripped_line, line_no} | adjusted_lines], 0, module)
+        adjust_indent(
```

Comment:
Please try to fit this horizontally, something like:

```elixir
    adjusted_lines = [{stripped_line, line_no} | adjusted_lines]
    adjust_indent(:text, rest, line_no + 1, adjusted_lines, 0, module)
```

Reply:
this will probably break the code because `adjusted_lines` is used in its original form a couple of lines below that. This is exactly the reason why I don't like re-assigning variables ;-)

What are your feelings towards the Erlang style of writing inline "comments" like `_indent = 0` instead of just `0`?

Reply:
> this will probably break the code because adjusted_lines is used in its original form a couple of lines below that

Sure, it was just a suggestion. I haven't considered the code fully.

It is OK to write inline comments but probably not as part of this PR so we keep it mostly around formatting.